component.json

```diff
@@ -91,6 +91,41 @@
           }
         }
       }
+    },
+    "delete": {
+      "main": "./lib/actions/delete.js",
+      "title": "Delete files",
+      "description": "Delete file by name in provided package",
```

Comment:
What `package` means in terms of the SFTP? Maybe better to replace it with `Delete file by path and filename`?